File: Server/ChatServer/src/main/java/com/szlazakm/chatserver/dtos/response/UserDTO.java
package com.szlazakm.chatserver.dtos.response;

import lombok.Builder;
import lombok.Value;
import lombok.extern.jackson.Jacksonized;

@Value
@Builder
@Jacksonized
public class UserDTO {

    String firstName;
    String lastName;
    String phoneNumber;
    String identityKey;
}

File: client/app/src/main/java/com/szlazakm/safechat/webclient/dtos/UserDTO.kt
package com.szlazakm.safechat.webclient.dtos

import com.fasterxml.jackson.annotation.JsonProperty

data class UserDTO(
    @JsonProperty("firstName")
    val firstName: String,
    @JsonProperty("lastName")
    val lastName: String,
    @JsonProperty("phoneNumber")
    val phoneNumber: String,
    @JsonProperty("identityKey")
    val identityKey: String,
)
